.rubocop.yml

```diff
@@ -0,0 +1,12 @@
+AllCops:
+  Exclude:
+    - "test/**/*"
+
+Metrics/LineLength:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
Pretty standard RuboCop config in this file (and in `.rubocop`) that I've used with great success on other projects. This file is customizable based on your preferences and I imagine we'll tweak this in the future.